### PR TITLE
Enable horizontal views

### PR DESCRIPTION
**Pull request by suhlrich**
We only needed to rotate the intrinsics. By re-writing the videos, they get automatically rotated, which is nice.

I tested:
- one vertical, one horizontal
- one leaning left, one leaning right
- TODO one upside down, one leaning left
- running the currently released ios app

The iphone portrait lock issue is non-intuitive. Posted issue to ios repo.

I think we can push this to the backend servers whenever. Let's see if we can get the error message on ios before pushing ios update.

### CODE REVIEW

File: utilsChecker.py [hunk 1]
```diff
@@ -280,7 +281,57 @@ def saveCameraParameters(filename,CameraParams):
...
+        elif rotation == 180: # leaning right
+            # Flip image size
+            CamParams['imageSize'] = np.flipud(CamParams['imageSize'])
+            # Flip focal lengths
+            CamParams['intrinsicMat'][0,0] = fy
+            CamParams['intrinsicMat'][1,1] = fx
+            # Change principle point - from upper left
+            CamParams['intrinsicMat'][0,2] = ly-py
+            CamParams['intrinsicMat'][1,2] = px
+        elif rotation == 279: # upside down
```
Comment (antoinefalisse) [inline utilsChecker.py:328]: Typo: should be `rotation == 270` I guess

**Review by antoinefalisse — CHANGES_REQUESTED** [review]
Looking good, one comment.

**Comment by antoinefalisse** [comment]
@suhlrich please also contribute to the changelog: https://github.com/stanfordnmbl/opencap-core/blob/main/CHANGELOG.md to keep track of all changes we are making (inspired from [opensim](https://github.com/opensim-org/opensim-core/blob/master/CHANGELOG.md)). Thanks.

**Comment by suhlrich** [comment]
Great. I'll merge into dev. Let's switch the servers over one by one. It is backward compatible, so no issue doing that.

**Pull request merged**